src/editors/containers/VideoEditor/components/VideoSettingsModal/ErrorSummary.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import { Alert } from '@edx/paragon';
+import { Info } from '@edx/paragon/icons';
+
+import messages from './components/messages';
+import { ErrorContext } from '../../hooks';
+import * as module from './ErrorSummary';
+
+export const hasError = (errHook) => (Object.keys(errHook[0]).length !== 0);
+
+export const ErrorSummary = () => {
+  const error = React.useContext(ErrorContext);
+  return (
+    <Alert
+      icon={Info}
+      show={Object.values(error).every(module.hasError)}
+      variant="danger"
+    >
+      <Alert.Heading>
+        <FormattedMessage {...messages.validateErrorTitle} />
+      </Alert.Heading>
+      <p>
+        <FormattedMessage {...messages.validateErrorBody} />
```

Comment:
Will this always be a fixed message or will it change based on the errors from the components? Like will it tell you which widgets have the errors? If not, does it make sense for the values to be booleans instead of objects?

Reply:
Based on the figma, it would always be a fixed message. I figured it should just pull in the error context to evaluate the boolean here. Normally, we add a hooks file, but there's not that much logic here.